From 1e75ecf2d0acb501bc63ef9cae1f0fde38cbdd75 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Thu, 24 Dec 2015 17:35:38 +0800
Subject: [PATCH] MLK-12076-1 ARM: dts: imx6ul: add "imx6ul-sdma"

Since ERR008517 workaound only fixed on i.mx6ul, we need extend one param
for ecspi sdma script to distinguish workaround needed or not in script.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 72fb004..cf9d854 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -733,7 +733,7 @@
 			};
 
 			sdma: sdma@020ec000 {
-				compatible = "fsl,imx6sx-sdma", "fsl,imx35-sdma";
+				compatible = "fsl,imx6ul-sdma", "fsl,imx35-sdma";
 				reg = <0x020ec000 0x4000>;
 				interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6UL_CLK_SDMA>,
-- 
1.8.0

